azure-mgmt-botservice/tests/test_mgmt_botservice_channels.py

```diff
@@ -0,0 +1,178 @@
+                    display_name = display_name,
+                    description= description,
+                    endpoint = endpoint,
+                    msa_app_id = msa_app_id,
+                    developer_app_insight_key = developer_app_insight_key,
+                    developer_app_insights_api_key = developer_app_insights_api_key,
+                    developer_app_insights_application_id = developer_app_insights_application_id,
+                )
+            )
+        )
+    
+    def tearDown(self):
+        super(BotServiceChannelsTestCase, self).tearDown()
+        self.client.bots.delete(
+            resource_group_name = self.resource_group_name,
+            resource_name = self.resource_name
+        )
+
+    def validateCreateGetAndDeleteChannel(self, channel_name, channel_properties):
+        self.createBot()
+
+        botChannel = BotChannel(
+            location = 'global',
+            properties = channel_properties
+        )
+
+        self.client.channels.create(
+            resource_group_name = self.resource_group_name,
+            resource_name = self.resource_name,
+            channel_name = channel_name,
+            parameters = botChannel 
+        )
+
+        channel = self.client.channels.get(
+            resource_group_name = self.resource_group_name,
+            resource_name = self.resource_name,
+            channel_name = channel_name
+        )
+        
+        self.assertIsNotNone(channel)
+        #is_enabled being true means that the service has managed to get the channel working.
+        if channel_name == 'DirectLineChannel':
+            self.assertTrue(channel.properties.properties.sites[0].is_enabled)
+        else:
+            self.assertTrue(channel.properties.properties.is_enabled)
+
+        channel = self.client.channels.delete(
+            resource_group_name = self.resource_group_name,
+            resource_name = self.resource_name,
+            channel_name = channel_name
+        )
+        with self.assertRaises(ErrorException):
+            channel = self.client.channels.get(
+                resource_group_name = self.resource_group_name,
+                resource_name = self.resource_name,
+                channel_name = channel_name
+            )
+
+    @ResourceGroupPreparer(name_prefix='pythonsdkbot')
+    def test_email_channel(self, resource_group):
+        self.resource_group_name = resource_group.name
+        from azure.mgmt.botservice.models import EmailChannel,EmailChannelProperties
+        channel = EmailChannel(
+            properties = EmailChannelProperties(
+                email_address = 'swagatm2@outlook.com',
+                password = 'Botuser123@',
+                is_enabled = True
+            )
+        )
+
+        self.validateCreateGetAndDeleteChannel(
+            channel_name = 'EmailChannel',
+            channel_properties = channel
+        )
+
+    # @ResourceGroupPreparer(name_prefix='pythonsdkbot')
```

Comment:
the test is currently failing in live because of issues with skype and teams channel services. i plan to uncomment them once those services are fixed and i can run the test . Or would you rather have this removed and added in a separate pr?